packages/polling-controller/src/StaticIntervalPollingController.ts

```diff
@@ -54,6 +68,13 @@ function StaticIntervalPollingControllerMixin<
         // TODO: Either fix this lint violation or explain why it's necessary to ignore.
         // eslint-disable-next-line @typescript-eslint/no-misused-promises
         async () => {
+          const durationForId = this.getDurationForId(key);
+          if (durationForId && Date.now() > durationForId) {
+            this._stopPollingByPollingTokenSetId(key);
```

Comment:
Is it necessary to call `_stopPollingByPollingTokenSetId` here? It seems that this is already called above, i.e., each time `_startPolling` is called, any existing poll is removed already. So maybe we can remove this line?

``` suggestion
```